guides/micronaut-static-resources/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
```

Comment:
remove this file. We don't need it. it would be generated.

Reply:
done 👍